pkg/dns/aws/dns.go

```diff
@@ -101,13 +101,26 @@ func NewProvider(config Config, operatorReleaseVersion string) (*Provider, error
 	if len(region) == 0 {
 		return nil, fmt.Errorf("region is required")
 	}
+	awsconfig := &aws.Config{}
+	awsconfig.Region = &region
+
+	// TODO: This API will only return hostedzone resources (which are global)
+	// when the region is forced to us-east-1. We don't yet understand why.
+	tagRegion := "us-east-1"
+
+	//If the region is in aws china, cn-north-1 or cn-northwest-1, we should:
+	// 1. hard code route53 api endpoint to https://api.route53.cn, as route53 is not GA in AWS China, and aws sdk didn't have the endpoint.
+	// 2. use the aws china regions to setup tagging api correctly in stead of "use-east-1"
```

Comment:
Typo `use-east-1`

Reply:
Also add a space after the `//` on L111.